test/integration/http_protocol_integration.h

```diff
@@ -6,11 +6,17 @@
 
 namespace Envoy {
 
+enum class Http2Impl {
+  Bare,
+  Wrapped,
+  Oghttp2,
+};
```

Comment:
Some comments here would be nice to have, not immediately clear what Bare vs Wrapped is

Reply:
I renamed the enum values to be clearer to casual readers. Let me know what you think!